dali/operators/generic/pad.h

```diff
@@ -107,6 +107,9 @@ class Pad : public Operator<Backend> {
     }
 
     // If a single *align* value is provided, use this value for all the axes
+    for (auto &align : align_){
```

Comment:
```suggestion
    for (auto &align : align_) {
```
The linter didn't protest?

Reply:
Done